welcome/about-hcp.adoc

```diff
@@ -65,10 +65,10 @@ Use the following sections to find content to help you learn about and use {hcp-
 | xref:../backup_and_restore/application_backup_and_restore/oadp-intro.adoc#oadp-api[Back up and restore]
 |
 |
-
-|
+//adding condition to get hcp upgrading PR built
+|ifdef::openshift-rosa-hcp[]
 | xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-hcp-upgrading[Upgrading]
-|
+|endif::openshift-rosa-hcp[]
```

Comment:
Same issue as the other "About HCP" topic, but I don't think this is in the HCP topic map.